packages/client/src/Call.ts

```diff
@@ -447,6 +447,9 @@ export class Call {
     this.state.setMetadata(response.call);
     this.state.setMembers(response.members);
 
+    this.watching = true;
```

Comment:
We should add `await httpClient.connectionIdPromise;`, and if we add watching to the `get` call we should also add the same logic to the `gotOrCreate` method